charon/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "charon"
-version = "0.1.20"
+version = "0.1.21"
```

Comment:
This is not necessary here. The CI check will complain, but it's a false positive since adding helpers cannot break users of charon-ml.